hack/runtime-migrator/internal/runtime/migrator.go

```diff
@@ -145,10 +145,26 @@ func getAdministratorsList(ctx context.Context, provider kubeconfig.Provider, sh
 	})
 
 	subjects := make([]string, 0)
-
 	for _, clusterRoleBinding := range clusterRoleBindings.Items {
-		for _, subject := range clusterRoleBinding.Subjects {
-			subjects = append(subjects, subject.Name)
+		// We are interested only in cluster-admin role
+		if clusterRoleBinding.RoleRef.Kind == "ClusterRole" && clusterRoleBinding.RoleRef.Name == "cluster-admin" {
+			willMigrate := false
+			for _, subject := range clusterRoleBinding.Subjects {
+				// We are interested only in users
+				if subject.Kind == rbacv1.UserKind {
+					subjects = append(subjects, subject.Name)
+					willMigrate = true
+				}
+			}
+
+			if willMigrate {
+				clusterRoleBinding.ObjectMeta.Labels["kyma-project.io/deprecation"] = "this ClusterRoleBinding is deprecated and will be removed in next days"
+				_, err := clientset.RbacV1().ClusterRoleBindings().Update(ctx, &clusterRoleBinding, metav1.UpdateOptions{})
```

Comment:
Important question: what would happen if KIM will take over the Runtime CR? Correct me, if I'm wrong but it will delete the CRB...